.rubocop.yml

```diff
@@ -603,6 +603,8 @@ Capybara/NegationMatcherAfterVisit: # new in 2.22
 Rails/MultipleRoutePaths: # new in 2.29
   Enabled: true
 Rails/StrongParametersExpect: # new in 2.29
-  Enabled: true
+  Enabled: false
```

Comment:
this rule when run with auto-correct was breaking the app by making changes to the params fetching in job_runs_controller and batch_context_controllers

Reply:
OK, no prob.